lib/Conversion/HWToLLVM/HWToLLVM.cpp

```diff
@@ -79,6 +79,38 @@ static Value zextByOne(Location loc, ConversionPatternRewriter &rewriter,
 // Extraction operation conversions
 //===----------------------------------------------------------------------===//
 
+namespace {
+/// Convert a StructExplodeOp to a LLVM dialect.
```

Comment:
Nit: to a LLVM dialect -> to the LLVM dialect

Reply:
Thanks! Didn't notice this little problem of improper terminology :smile: